### PR TITLE
[8.0] skip invoice lines with value 0

### PR DESCRIPTION
**Pull request by luc-demeyer**

### CODE REVIEW

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test.

**Comment by luc-demeyer** [comment]
@cubells
I changed the check on 'price_unit' with value 0.0 into 'price_subtotal'.
By doing so we also cover the case of invoice lines with 100% discount.
Let me know if it's ok so that we can merge (and forward port to v10)

File: intrastat_product/models/intrastat_product_declaration.py [hunk 1]
```diff
@@ -501,6 +501,9 @@ def _gather_invoices(self):
             total_inv_weight = 0.0
             for inv_line in invoice.invoice_line:
 
+                if not inv_line.price_subtotal:
+                    continue
+
```
Comment (cubells) [inline intrastat_product/models/intrastat_product_declaration.py]: Remove all the white spaces of this line to avoid lint errors.

**Review by cubells — APPROVED** [review]
@luc-demeyer 

If you remove white spaces all is ok for me.

File: intrastat_product/models/intrastat_product_declaration.py (hunk 1 above)
Comment (pedrobaeza) [reply intrastat_product/models/intrastat_product_declaration.py]: A better expression is:
```python
for inv_line in invoice.invoice_line.filtered('price_subtotal'):
    ...
```

**Comment by pedrobaeza** [comment]
Travis is red

**Pull request merged**